pages/features.html

```diff
@@ -493,7 +493,9 @@ <h4>Leverage your C# experience to feel right at home</h4>
 				for the .NET platform. Power your game with familiar libraries and give
 				them performance boost, while still benefiting from close engine integration.
 				<br><br>
-				<strong>Note:</strong> .NET support is provided as a dedicated engine executable. C# support is currently only available for desktop platforms in Godot 4. Use <a href="/download/3.x/">Godot 3</a> to run C# on Android, iOS and Web.
+				<strong>Note:</strong> .NET support is provided as a dedicated engine executable.
+				C# support is currently only available for desktop and mobile platforms as of Godot 4.2.
+				Use <a href="/download/3.x/">Godot 3</a> to run C# on the Web platform.
```

Comment:
```suggestion
				C# support is available for desktop and mobile platforms as of Godot 4.2.
				Web support should be added in the future, but until then <a href="/download/3.x/">Godot 3</a> is still a supported option.
```

Reply:
For context, not sure about the flow of the second sentence, but I felt the first sentence about which platforms are supported should be formulated positively.